src/main/java/org/openelisglobal/samplehuman/daoimpl/SampleHumanDAOImpl.java

```diff
@@ -145,7 +145,7 @@ public List<Sample> getSamplesForPatient(String patientID) throws LIMSRuntimeExc
                     + " sampleHuman.sampleId = sample.id and sampleHuman.patientId = :patientId order by"
                     + " sample.id";
             Query<Sample> query = entityManager.unwrap(Session.class).createQuery(sql, Sample.class);
-            query.setParameter("patientId", Integer.parseInt(patientID));
```

Comment:
`sampleHuman.patientId` is stored as an Integer in the DB . so revert this comment

Reply:
But I was getting an error indicating otherwise in my test

Reply:
Anyways let me me revert so that we can see